src/providers/hana/qgshanaresultset.cpp

```diff
@@ -0,0 +1,142 @@
+/***************************************************************************
+   qgshanaresultset.h
+   --------------------------------------
+   Date      : 31-05-2019
+   Copyright : (C) SAP SE
+   Author    : Maxim Rylov
+ ***************************************************************************/
+
+/***************************************************************************
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ ***************************************************************************/
+#include "qgshanaresultset.h"
+#include "qgshanautils.h"
+#include "qgslogger.h"
+#include <QString>
+
+QgsHanaResultSet::QgsHanaResultSet( ResultSetRef &&resultSet )
+  : mResultSet( resultSet )
```

Comment:
You have to use `std::move()` here to move `resultSet`.

Reply:
Done in 4d36f6f.